lib/ansible/modules/system/cron.py

```diff
@@ -705,6 +705,11 @@ def main():
                 crontab.remove_env(name)
                 changed = True
     else:
+        # Remove trailing new lines to properly compare job content after
+        # The `old_job` always got newline stripped due to nature of cron files
+        # But it's common to put a very long `job` into several lines with `key: >` or `key: |` YAML syntax
+        job = job.rstrip('\n')
```

Comment:
I'm surprised this is necessary after ansible/ansible-modules-core#4285, where [we already strip leading/trailing whitespace in the `get_cron_job` call](https://github.com/PikachuEXE/ansible/blob/c32bb88df56aa5e7972fa6a72358de5b31f00dad/lib/ansible/modules/system/cron.py#L448)